meshroom/ui/qml/GraphEditor/AttributeItemDelegate.qml

```diff
@@ -193,16 +207,29 @@ RowLayout {
                 }
             }
 
+            MaterialLabel {
+                property bool isDisplayable: attribute.isOutput && (attribute.is2D || attribute.is3D)
+                property bool isDisplayed: attribute === _reconstruction.displayedAttr2D || _reconstruction.displayedAttrs3D.count && _reconstruction.displayedAttrs3D.contains(attribute)
+                text: isDisplayed ? MaterialIcons.visibility : MaterialIcons.visibility_off
+                enabled: isDisplayed
+                visible: isDisplayable
+
+                Layout.alignment: Qt.AlignRight
+                rightPadding: 5
```

Comment:
The alignment on the right is not needed here since the label cannot move freely, so I'd remove it. Same for the top padding, as the label seems to be vertically centered by default. 

However, the "advanced" button and the navigation tool buttons have a global padding of 4 pixels, so for the sake of harmony, this one should also be set to 4.

```suggestion
                padding: 4
```